client.js

```diff
@@ -8,30 +7,19 @@ import { Submarine } from './src/submarine';
 //
 // See https://hub.getsubmarine.com/docs/the-submarine-customer-api#authentication for how
 // Submarine Customer API signatures should be generated using the secret in a safe way.
-const DEV_ENVIRONMENT = 'staging';
-const DEV_CUSTOMER_ID = '6211133636859';
-const DEV_SHOP_DOMAIN = 'submarine-js.myshopify.com';
-const DEV_API_SECRET = '1YoSfSMxd6LHHSdbGyp8yTBQ';
-
-// Calculate an API signature for the given customer, timestamp and API secret.
-const calculateSignature = (customerId, timestamp, secret) => {
-  const data = `${customerId}:${timestamp}`;
-  return sha256.hmac(secret, data);
-};
+const DEV_ENVIRONMENT = 'dev';
```

Comment:
@appu4ever lets just leave the original staging placeholder creds as they were, rather than hardcoding values specific to your dev environment.